From a2a02c4fb2029bb1ec65699f6a27cdc22fbd48ff Mon Sep 17 00:00:00 2001
From: Linh Hua <linh.hua.jc@renesas.com>
Date: Tue, 4 Mar 2025 10:40:10 +0700
Subject: [PATCH 3/3] configs: rzf_defconfig: enable ECC Detection mode support
 for RZ/Five

Enable ECC Detection mode support for RZ/Five SMARC and Development EVK
DDR4 Boards.

Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
Signed-off-by: Linh Hua <linh.hua.jc@renesas.com>
---
 configs/rzf-dev_ddr4_defconfig | 1 +
 configs/smarc-rzf_defconfig    | 1 +
 2 files changed, 2 insertions(+)

diff --git a/configs/rzf-dev_ddr4_defconfig b/configs/rzf-dev_ddr4_defconfig
index 320ae8b018..2ef7fd3e62 100644
--- a/configs/rzf-dev_ddr4_defconfig
+++ b/configs/rzf-dev_ddr4_defconfig
@@ -145,3 +145,4 @@ CONFIG_SBI_V02=y
 CONFIG_CMD_EXCEPTION=y
 CONFIG_CMD_RZG2L_ECC=y
 CONFIG_RZF_DDR_ECC=y
+CONFIG_RZF_DDR_ECC_DETECT=y
diff --git a/configs/smarc-rzf_defconfig b/configs/smarc-rzf_defconfig
index bfa053662a..50b04cf75e 100644
--- a/configs/smarc-rzf_defconfig
+++ b/configs/smarc-rzf_defconfig
@@ -170,3 +170,4 @@ CONFIG_USB_FUNCTION_MASS_STORAGE=y
 CONFIG_CMD_USB_MASS_STORAGE=y
 CONFIG_CMD_RZG2L_ECC=y
 CONFIG_RZF_DDR_ECC=y
+CONFIG_RZF_DDR_ECC_DETECT=y
-- 
2.17.1

